linera-views/src/reentrant_collection_view.rs

```diff
@@ -526,11 +550,8 @@ where
         }
         let response = futures::future::join_all(handles).await;
         for (i, view) in response.into_iter().enumerate() {
-            let (short_key, to_be_cleared) = &selected_short_keys[i];
-            let mut view = view??;
-            if *to_be_cleared {
-                view.clear();
-            }
+            let short_key = &selected_short_keys[i];
```

Comment:
nit: could be inlined

Reply:
inlined was not a great idea, but I could do a nice rewrite.

Reply:
Yes, I could remove out the index and clean up the code.